features/rules/ALB/ALB031_Alignment-layouts-default-case.feature

```diff
@@ -0,0 +1,18 @@
+@implementer-agreement
+@ALB
+@version1
+Feature: ALB031 - Alignment Layouts Default Case
+The rule verifies that alignments are nested in accordance with Concept Template 4.1.4.4.1.1
+which represents the default case for nesting alignment layouts.
+Alignment layouts are abbreviated as follows: horiz = IfcAlignmentHorizontal, vert = IfcAlignmentVertical,
+and cant = IfcAlignmentCant
+
+  Scenario: Agreement on IfcAlignment layout nesting per 4.1.4.4.1.1.
+
+      Given A model with Schema 'IFC4.3'
+      Given an .IfcAlignment.
+      Given .IsDecomposedBy. ^is^ empty
+      Given its attribute .IsNestedBy.
```

Comment:
I think it reads a bit clearer if you use the relationship predicate here, either:

(a) check for existance

```suggestion
    Given a relationship .IfcRelNests. from .IfcAlignment. to .IfcObject.
```

(b) check for existence and move focus

```suggestion
    Given a relationship .IfcRelNests. from .IfcAlignment. to .IfcObject. and following that
```